Examples/Podfile.lock

```diff
@@ -0,0 +1,41 @@
+PODS:
+  - AFNetworking (3.2.1):
+    - AFNetworking/NSURLSession (= 3.2.1)
+    - AFNetworking/Reachability (= 3.2.1)
+    - AFNetworking/Security (= 3.2.1)
+    - AFNetworking/Serialization (= 3.2.1)
+    - AFNetworking/UIKit (= 3.2.1)
+  - AFNetworking/NSURLSession (3.2.1):
+    - AFNetworking/Reachability
+    - AFNetworking/Security
+    - AFNetworking/Serialization
+  - AFNetworking/Reachability (3.2.1)
+  - AFNetworking/Security (3.2.1)
+  - AFNetworking/Serialization (3.2.1)
+  - AFNetworking/UIKit (3.2.1):
+    - AFNetworking/NSURLSession
+  - JSONModel (1.8.0)
+  - VelocidiSDK (0.0.1):
+    - AFNetworking (~> 3.2.1)
+    - JSONModel (~> 1.8)
+
+DEPENDENCIES:
+  - VelocidiSDK (from `../`)
+
+SPEC REPOS:
+  https://github.com/cocoapods/specs.git:
+    - AFNetworking
+    - JSONModel
+
+EXTERNAL SOURCES:
+  VelocidiSDK:
+    :path: "../"
+
+SPEC CHECKSUMS:
+  AFNetworking: b6f891fdfaed196b46c7a83cf209e09697b94057
+  JSONModel: 02ab723958366a3fd27da57ea2af2113658762e9
+  VelocidiSDK: 498898c7453b867bfd6f8f288f9b603676ef08d5
+
+PODFILE CHECKSUM: ba8278e89e7a7f65f231b280ae9ba6dfdc615d4a
+
+COCOAPODS: 1.6.2
```

Comment:
Cocoapods is currently at 1.7.0. Can we generate the lock file with the latest version?

Reply:
Updated it in 5aa7b31